inductiva/logs/task_logs.py

```diff
@@ -0,0 +1,50 @@
+"""Module for streaming the logs of a task through a websocket."""
+import sys
+
+import websocket
+
+from inductiva import constants
+
+
+class TaskLogsStream:
+    """Logs the streams of a running task through a websocket."""
+
+    def __init__(self, task_id, fout=sys.stdout, ferr=sys.stderr):
+        """Initialize the websocket connection to the task's logs.
+
+        Args:
+            task_id (int): ID of the task to print the logs of."""
+
+        self.websocket_url = f"{constants.LOGS_WEBSOCKET}/{task_id}"
+        self.task_id = task_id
+        self.fout = fout
+        self.ferr = ferr
+        self.ws = self.setup_websocket()
+
+    def setup_websocket(self):
+        """Initialize the websocket app, with the callbacks within the class."""
+        return websocket.WebSocketApp(self.websocket_url,
+                                      on_open=self.on_open,
+                                      on_message=self.on_message,
+                                      on_error=self.on_error,
+                                      on_close=self.on_close)
+
+    def on_message(self, ws, message):  # pylint: disable=unused-argument
```

Comment:
Why the extra unused arguments? I would expect this if `TaskLogsStream` was inheriting from another class. But that does not seem to be the case

Reply:
The `ws` is the websocket connection, which a requirement for the respective attribute method in the `WebSocketApp`.

Reply:
can you explain better? I have the same doubt as @AugustoPeres ...
I think pylint was alerting to the fact that you are not using the argument "ws" anywhere in the code -- and ideally we would not disable that warning because it's there for a good reason... Like Augusto said, it seems like if this was derived class from a base class where those methods were declared (and it had those arguments) then ok, but your class does not seem to be inheriting from any base class... So can't you just get rid of it?
If not (because it's indeed implementing a method from a base class) and for some reason you don't use an argument, there are typically a few ways to avoid a linting warning in python (without disabling pylint):
1) delete the variable 
`del myvar`
2) prepend the name with a standard prefix that pylint understand, typically is `unused_` so it would be `unused_myvar` or sometimes is underscore `_myvar`...
See a long thread here:
https://stackoverflow.com/questions/10025680/how-can-i-denote-unused-function-arguments

Reply:
This method is a callback. It needs to comply with the contract expected by the WebSocketApp. It is not used but needs to be there. This being said, I agree with the renaming of the unused variable